pkg/cli/update.go

```diff
@@ -0,0 +1,151 @@
+package cli
+
+import (
+	"archive/zip"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"syscall"
+
+	"github.com/google/go-github/github"
+	"github.com/sosedoff/pgweb/pkg/command"
+)
+
+var (
+	latestVersion string
+	workDir       = "/tmp"
+	distName      = fmt.Sprintf("pgweb_%s_%s", runtime.GOOS, runtime.GOARCH)
+	zipName       = distName + ".zip"
+)
+
+func checkUpdate() {
+
+	if options.DisableCheckUpdate {
+		return
+	}
+
+	client := github.NewClient(nil)
+
+	latestRelease, _, err := client.Repositories.GetLatestRelease(context.Background(), "sosedoff", "pgweb")
+	if err != nil {
+		fmt.Errorf("GitHub is unavailable")
+	}
+
+	latestVersion = *latestRelease.Name
+
+	if latestVersion != command.VERSION {
+		var upgrade string
+
+		fmt.Printf("A new version %s is available. Do you want to upgrade? [y/N]", latestVersion)
+		chars, err := fmt.Scanln(&upgrade)
```

Comment:
This change will block the server from starting until the user types Y/N. 
We could try and figure out if it's possible to do a background update until the next start, which is
a definitely better UX here.

Reply:
Because of that I've added --disable-check-update CLI option

Reply:
>We could try and figure out if it's possible to do a background update until the next start, which is
>a definitely better UX here.

You know this an idea.